src/org/labkey/test/tests/SpecimenImportTest.java

```diff
@@ -116,7 +117,7 @@ protected void changeTimepointType()
         clickTab("Manage");
         waitAndClick(Locator.linkContainingText("Change Study Properties"));
         waitForText("Timepoint Type:");
```

Comment:
Your fix seems more effective than this `waitForText`. There's also a page load here, I think.
```suggestion
        waitAndClickAndWait(Locator.linkContainingText("Change Study Properties"));
```